packages/devextreme-scss/scss/widgets/generic/gridBase/_index.scss

```diff
@@ -32,6 +32,7 @@ $generic-grid-base-checkbox-icon-size: 16px;
 $generic-grid-base-checkbox-indeterminate-icon-size: 6px;
 $grid-masterdetail-padding: 20px;
 $generic-grid-base-group-panel-message-line-height: $generic-button-text-line-height;
+$generic-grid-base-cell-input-height: round($generic-base-line-height * $generic-base-font-size) + 2 * $generic-grid-base-cell-padding;
```

Comment:
What is `2`? Would be great to have variable with appropriate name which describes the idea of the value, it will be especially useful for future code support.

Reply:
I extracted the variable with appropriate name [from here](https://github.com/DevExpress/DevExtreme/pull/28412/files#diff-75c4d4384baf83c83521cc9470a87074164eafb958d7ffd002f8a2f91768aa06L243). What is wrong?

Reply:
before you described, it was not clear what is 2, padding, or border or something else. I'm sure that after 1 month you would not recollect what does it mean. Please create variable.

Reply:
All other things LGTM to me.